Sources/AIProxy/Fal/FalRunwayGen3AlphaOutputSchema.swift

```diff
@@ -0,0 +1,34 @@
+//
+//  FalRunwayGen3AlphaOutputSchema.swift
+//
+//
+//  Created by Todd Hamilton on 10/4/24.
+//
+
+import Foundation
+
+public struct FalRunwayGen3AlphaOutputSchema: Decodable {
+    public let video: Video?
+
+    private enum CodingKeys: String, CodingKey {
+        case video
+    }
+}
+
+extension FalRunwayGen3AlphaOutputSchema {
+    public struct Video: Decodable {
+        public let url: URL?                // The URL where the file can be downloaded from.
```

Comment:
Write these as doctrings with triple slash above the property, that way Xcode's help text will display the comment when you option-tap on the property from a call site